language/en-GB/en-GB.com_joomgallery.ini

```diff
@@ -659,4 +646,41 @@ COM_JOOMGALLERY_CONFIG_GS_TAB_BACKEND_REPLACEVALUES_IPTC="IPTC"
 ;-------
 ;Changed
 ;-------
-COM_JOOMGALLERY_UPLOAD_BATCH_UPLOAD_NOTE="Please do not upload zipped directories!<br />Only zipped image files will be stored correctly.<br />Note that there is no security check for zip files. Only upload zip files if you are sure that the zip files contain only secure image files!"
\ No newline at end of file
+COM_JOOMGALLERY_UPLOAD_BATCH_UPLOAD_NOTE="Please do not upload zipped directories!<br />Only zipped image files will be stored correctly.<br />Note that there is no security check for zip files. Only upload zip files if you are sure that the zip files contain only secure image files!"
+;-------
+;Changed/Added with the PR "JoomIMGtools"
+;-------
+COM_JOOMGALLERY_UPLOAD_CREATE_THUMBNAIL_FROM="Resize image from"
+COM_JOOMGALLERY_UPLOAD_RESIZE_TO_MAX="Resizing to max edge lenght using "
```

Comment:
Check spelling: leng**th**

Reply:
Thanks!
Seems that the constant COM_JOOMGALLERY_UPLOAD_CREATE_THUMBNAIL_FROM first has been changed by you but now is not used anymore. You should move it with it' original value to section of the removed constants.